File: laboratories/ppo/exercises/lab07/python/main.py
from src.vehicles.Bike import Bike
from src.vehicles.Car import Car


vehicles = [
    Car("DL00001"),
    Bike("John Doe"),
    Bike(),
    Car("DL00002", "Jane Doe"),
    Car("DLU0001"),
]

for vehicle in vehicles:
    print(vehicle.identify())

File: laboratories/ppo/exercises/lab07/java/Main.java
import lab07.Vehicles.Bike;
import lab07.Vehicles.Car;
import lab07.Vehicles.Vehicle;

public class Main {
  public static void main(String[] args) {
	Vehicle[] vehicles = {
        new Car("DL00001"),
        new Bike("John Doe"),
        new Bike(),
        new Car("DL00002", "Jane Doe"),
        new Car("DLU0001"),
	};

	for(Vehicle vehicle : vehicles) {
		System.out.println(vehicle.identify());
	}
  }
}
